src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -866,10 +921,22 @@ protected function _BuildFilterSection()
 
         if ($this->showSearchBox) {
             $this->somethingToShow = true;
+
+            $filterContent .= '<div class="form-group mr-2">';
+            $filterContent .= '<select id="searchLookup" name="_search_word" data-listname="'. TGlobal::OutHTML($this->listName) .'" class="form-control" data-select2-placeholder="';
+            $filterContent .= $this->searchFieldText;
+            $filterContent .= '" data-select2-ajax="'. $this->getAjaxRecordsUrl() .'">';
```

Comment:
Maybe we should escape this value.

Reply:
I'm pretty sure, we had this first, but escaping it here broke the URL for the selec2 plugin. Maybe OutHTML was not the right method here. But we should try this first.

Reply:
it works with OutHTML, so I used it for $this->getAjaxRecordsUrl()